Set/Game/GameView.swift

```diff
@@ -9,9 +9,16 @@ struct GameView: View {
             AspectVGrid(items: gameViewModel.cardsOnScreen, aspectRatio: 2 / 3) { card in
                 cardViewBuilder.build(for: card)
                     .onTapGesture {
-                        gameViewModel.select(card)
+                        guard let player = gameViewModel.whoseTurn else {
+                            return
+                        }
+                        gameViewModel.select(card, player)
                     }
             }
+            .onReceive(gameViewModel.timer) { _ in
```

Comment:
View should not know anything about timer, please make `timer` private in `ViewModel`

Reply:
fixed